keycloak/keycloak-admin-ui
# Grouped multiple credentials

## Motivation
Creating expandable rows for grouped credentials as per design https://marvelapp.com/prototype/a9b33db/screen/83342106

Implementation:
<img width="1657" alt="Screenshot 2021-12-13 at 16 46 23" src="https://user-images.githubusercontent.com/4890675/145854412-765a8770-3c01-4c25-a307-87a82764c18c.png">


**Note: I may need to fix some of the CSS for expandable rows as they don't seem to be very responsive at the moment** 

## Verification Steps
1. Provided that you have multiple credentials set up. 
2. Go to /users/credentials tab and verify that the rows are expandable

## Checklist:
